dev-docs/howtoguides/testing.md

```diff
@@ -57,36 +57,33 @@ The testing can be overridden to install ubuntu-advantage-tools from other sourc
 > **Note**
 > Note that we cache the source when running with `UACLIENT_BEHAVE_INSTALL_FROM=local` based on a hash, calculated from the repository state. If you change the python code locally and run the behave tests against your new version, there will be new debs in the cache source with the new repo state hash.
 
-To run the tests, you can use `tox`:
+To run the integration tests, use the `tox` command as shown below. 
+Please note that this command will execute all the integration tests sequentially and may take a significant amount of time to complete, depending on the number and complexity of the tests.
```

Comment:
This command as written will always take a long time - no "depending..." :)

```suggestion
Please note that, as shown here without arguments, this command would execute all the integration tests sequentially and would take an inordinate amount of time to complete. Always pass arguments to this command to specify a subset of tests to run, as demonstrated next. 
```

Reply:
Updated